x11-apps/radeontop/radeontop-1.0.ebuild

```diff
@@ -14,18 +14,19 @@ KEYWORDS="~amd64 ~x86"
 IUSE="nls"
 
 RDEPEND="
-	sys-libs/ncurses
-	x11-libs/libpciaccess
+	sys-libs/ncurses:0=
 	x11-libs/libdrm
-	nls? ( sys-libs/ncurses[unicode] virtual/libintl )
+	x11-libs/libpciaccess
+	x11-libs/libxcb
+	nls? ( sys-libs/ncurses:0=[unicode] virtual/libintl )
 "
 DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	nls? ( sys-devel/gettext )
 "
 
 src_prepare() {
-	epatch_user
+	default
 
 cat > include/version.h << EOF || die
```

Comment:
use the tab-version of cat: http://stackoverflow.com/questions/19734083/indenting-code-when-redirecting-text
this makes everything more readable and properly indented